### ISSUE TITLE
Minimal mesangial lupus nephritis

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6832]
**Preferred term label:**
Minimal mesangial lupus nephritis

**Synonyms**
Class I lupus nephritis

**Definition (free text, please give PubMed ID)**
A subclass of lupus nephritis with normal glomeruli by light microscopy, but mesangial immune deposits by immunofluorescence.

**Parent term (use hpo.jax.org/app)**
Lupus nephritis (newly proposed term)

**Related terms**
Mesangial Immune complex deposition HP:0033295
Systemic lupus erythematosus HP:0002725

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM #152700, ORPHA:536

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
added
